client/src/game/map.js

```diff
@@ -574,6 +578,13 @@ class Map extends EventEmitter {
 
     this.stars.forEach(s => s.refreshZoom(zoomPercent))
     this.carriers.forEach(c => c.refreshZoom(zoomPercent))
+
+    /*
```

Comment:
Is this code commented out for a reason? Remove if not needed.

Reply:
I think this logic has been moved into the `refreshZoom` function in `territories` so can probably be removed.

Reply:
i don't remember if that was a random fix i had siting on this branch or what was the purpose of this, i removed it anyway